### PR TITLE
Release v5.1.0

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

https://folio-org.atlassian.net/browse/UIORGS-426
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -351,8 +351,8 @@
     "regenerator-runtime": "^0.13.3"
```
Comment (UladzislauKutarkin) [inline package.json:351]: update version of app

Comment (alisher-epam) [reply package.json:351]: Can you be specific? What version and why? Did you update any packages recently?

**Comment by alisher-epam** [comment]
The build is failing due to the changes have not been merged yet https://github.com/folio-org/ui-plugin-find-organization/pull/124

File: package.json (hunk 1 above)
Comment (UladzislauKutarkin) [reply package.json:351]: I mean ui-organizations

Comment (UladzislauKutarkin) [reply package.json:351]: Oh it's already updated

**Review by UladzislauKutarkin — APPROVED** [review]

**Review by zburke — CHANGES_REQUESTED** [review]
* The PR title is "Release v5.1.0" but the CHANGELOG comment is "5.2.0 (IN PROGRESS)". 
* The CHANGELOG comment is "5.2.0 (IN PROGRESS)" but the version in `package.json` is still `5.1.0`
* There are changes to dependency versions in `package.json` that are unrelated to either publishing a release or bumping the version in preparation for the next release. 

You need to resolve the contradictions, and then handle these separate activities in separate PRs. You may find it easiest to close this PR and start over once the version updates are made but it's up to you. Make sure to commit the dependency changes first so they can be included in your `5.1.0` release.

**Comment by alisher-epam** [comment]
@zburke You are right. I will exclude the next release preparation steps from the PR.

**Pull request closed without merging**